sysutils/terragrunt/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+name                terragrunt
+version             0.23.4
+depends_run         port:terraform-0.12
```

Comment:
I suppose it could use `terraform-0.11` (or a newer version in the future) as well? You could probably do so (now or at a later point) through the use of variants, but leaving it like this for now is fine.

Reply:
I don't think this is the case as Terraform 0.12 introduced breaking changes to its syntax. Hence I believe 0.12 is required from Terragrunt 0.19 onward (which first accommodated this change). The general `terraform` port is marked obsolete and recommends the `terraform-0.12` port. I'll have to study the reference manual a bit more as this is my first time porting software so any further hints are welcome.

Reply:
you can certainly leave it like this, especially if it doesn't support the older (0.11) version. One can always think about supporting more versions if upstream does so as well.